### DISCUSSION TITLE
How to run UPX when releasing only?

### DISCUSSION THREAD

**Discussion by bgr** [goreleaser/goreleaser#4268] [Q&A] [answered]
I spent quite a bit of time trying out various combinations of syntax like the following:

    upx:
      - enabled: "{{ not .IsSnapshot }}"

but I couldn't figure it out, it's always an error like ```cannot unmarshal !!str `{{...` into bool```.

Is there a way to do this, or is using such syntax not valid for that field?
Is there another way to have UPX disabled for local builds while developing?

**Comment by caarlos0** [comment, marked as answer]
It isn't possible right now, just implemented it though: https://github.com/goreleaser/goreleaser/pull/4269


Will be available in the next release.
